api/segments/models.py

```diff
@@ -204,6 +212,16 @@ def _check_percentage_split_operator(self, identity):
             <= float_value
         )
 
+    def _check_modulo_operator(self, value: typing.Union[int, float]) -> bool:
```

Comment:
Seems like we have no test coverage for this logic. Can we add some? 

The following should give you an example of what we need: https://github.com/Flagsmith/flagsmith/blob/main/api/tests/unit/segments/test_conditions.py